compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs

```diff
@@ -911,9 +913,12 @@ renderScenarioResult world res = TL.toStrict $ Blaze.renderHtml $ do
             H.script "" H.! A.src "$webviewSrc"
         let tableView = renderTableView world res
         let transView = renderTransactionView world res
+        let noteView = H.div H.! A.class_ "note" H.! A.id "note" $ H.toHtml $ T.pack " "
         case tableView of
-            Nothing -> H.body transView
-            Just tbl -> H.body H.! A.class_ "hide_archived hide_transaction" $ do
+            Nothing -> H.body H.! A.class_ "hide_note" $ do
```

Comment:
Do we actually need the `hide_note` class? It looks like we are actually clearing the content so I don’t see why we ever need to hide it.

Reply:
I left it as is, since it's hiding whole `div` which is a container for notes' HTML. If shown, even when empty it could add some margins, etc, so it's better to have this hidden if not used.

Reply:
:+1: